### DISCUSSION TITLE
Dropping unmatched character '𝄞' (U+1d11e)

### DISCUSSION THREAD

**Discussion by perdakovich** [polybar/polybar#3061] [Q&A] [answered]
Links: 
https://github.com/polybar/polybar/issues/392#issuecomment-1858982813
https://github.com/polybar/polybar/discussions/2871#discussion-4624442

I use Arch Linux. I am using the mpd module with the example config from the [Polybar wiki](https://github.com/polybar/polybar/wiki/Module:-mpd). This is the error I have:
```
warn: Dropping unmatched character '𝄞' (U+1d11e) in ' 𝄞 Escape From Midwich Valley'
warn: Dropping unmatched character '🔁' (U+1f501) in '🔁'
warn: Dropping unmatched character '🔀' (U+1f500) in '🔀'
warn: Dropping unmatched character '𝄞' (U+1d11e) in ' 𝄞 Escape From Midwich Valley'
warn: Dropping unmatched character '🔁' (U+1f501) in '🔁'
warn: Dropping unmatched character '🔀' (U+1f500) in '🔀'
warn: Dropping unmatched character '𝄞' (U+1d11e) in ' 𝄞 Escape From Midwich Valley'
```

I have this in my polybar config:
```
font-0 = "Iosevka Nerd Font Propo,Iosevka NFP Medium:style=Medium,Regular:pixelsize=14;2
font-1 = "Symbols Nerd Font:style=Regular"
font-2 = "FontAwesome:style=Regular"
font-3 = "Siji:style=Regular:pixelsize=10;1"
```

Output of `fc-list | grep siji`: (I tried copying to the `.local/share/fonts` too)
```
/usr/share/fonts/misc/siji.bdf: Siji:style=Regular
/home/user/.local/share/fonts/siji.bdf: Siji:style=Regular
```

There was no `no-bitmaps` conf. 
I even manually added the `yes-bitmaps` conf, still no luck.

**Comment by patrick96** [comment, marked as answer]
Those characters are available in [unifont](https://unifoundry.com/unifont/index.html) (you need both the regular unifont and the "upper" variant). I thought we had that in the wiki that you should have unifont as a fallback, but I can't find it anymore. I'll update the wiki

**Comment by perdakovich** [comment]
Thanks, Unifont worked, but I wanted to go further:
I figured out that Firefox was using Noto Music for this symbol, so I imported Noto Music font into Polybar and now it shows just the same as in Firefox.

Thanks for the help!
